internal/cli/atlas/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/kubernetes/config"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "kubernetes"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: cli.GenerateAliases(use),
```

Comment:
similar to `config`, GenerateAliases here will generate only `kubernete` as an alias which may not be what you want and I think we discussed `k8s` as an alias

Reply:
Fixed